tests/rptest/services/redpanda.py

```diff
@@ -1207,7 +1204,10 @@ def clean(self, **kwargs):
         if self._s3client:
             self.delete_bucket_from_si()
 
-    def clean_node(self, node, preserve_logs=False, clean_installs=True):
+    def clean_node(self,
+                   node,
+                   preserve_logs=False,
+                   preserve_current_install=False):
```

Comment:
Now that i understand more, is this an option because when running in CDT it would otherwise not really be possible to do without manual intervention? Either case the comment still makes sense, the downloaded binaries should always be identical. Maybe cleaning the different versions of `HEAD`?

Reply:
Not quite, `preserve_current_install` exists because we run `clean_node()` before starting a Redpanda node, and we may have just installed new binaries on that node. In that case, we want to preserve what may have been install in the currently-running test

Note that this doesn't clean out any downloads, it just changes whether we change what `/opt/redpanda` points to

Reply:
Done